registrations/management/commands/report_pmtct_risks.py

```diff
@@ -115,7 +122,7 @@ def add_to_result(risk, reg):
                 validated=True)
 
             for registration in registrations.iterator():
-
+                add_personally_identifiable_fields(registration)
```

Comment:
Yup, can do it. It's a little complicated since it's not a task, but I wanted it in the same file as the remove personal fields task, since they're related.